pkg/bicep/bicep.go

```diff
@@ -6,11 +6,17 @@ import (
 	"path"
 
 	"github.com/massdriver-cloud/mass/pkg/bundle"
+	"github.com/massdriver-cloud/mass/pkg/files"
 	"github.com/massdriver-cloud/mass/pkg/inputvars"
 	"github.com/spf13/afero"
 )
 
-const paramsFile = "template.parameters.json"
+const ParamsFile = "template.parameters.json"
```

Comment:
I don't see this package variable reference elsewhere. Is there a reason it was made public?